### PR TITLE
update pdfcomprezzor to the latest version

### PR DESCRIPTION
**Pull request by christian-heusel**
* **What kind of change does this PR introduce?**
update

* **Please check if the PR fulfills these requirements**
   - [ ] E2E Tests for the changes have been added via Cypress
   - [ ] Meaningful rspec tests have been added
   - [ ] Docs have been added / updated 

* **What is the current behavior?** (You can also link to an open issue here)
pdfcomprezzor is outdated (fixes https://github.com/MaMpf-HD/mampf/issues/430)

* **What is the new behavior (if this is a feature change)?**
pdfcomprezzor has been updated (in its own repo) and was bumped here to the resulting latest version

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)
Hopefully not, but this need more testing by the project team. I only verified that the functunality itself still works.

* **Other information**:
🚫

### CODE REVIEW

**Comment by christian-heusel** [comment]
Maybe wait on https://github.com/henrixapp/pdfcomprezzor/pull/9, as a new version of pdfcpu was just released an hour ago 🤔

**Comment by christian-heusel** [comment]
> Maybe wait on [henrixapp/pdfcomprezzor#9](https://github.com/henrixapp/pdfcomprezzor/pull/9), as a new version of pdfcpu was just released an hour ago thinking

The new pdfcpu release (with the rather comical timing) has been pulled in and tested! ✅  
This is now ready for review! 🥳

**Comment by christian-heusel** [comment]
I found a few problems, the code currently has own docker files to build `henrixapp/pdfcomprezzor`, see the linked code below.
These references either need to be updated (which is ugly) or somehow use the dockerfile from `henrixapp/pdfcomprezzor`.

https://github.com/MaMpf-HD/mampf/blob/b211e4dcd37111a2dacd241668ad764c790bdae6/docker/run_cypress_tests/Dockerfile#L1-L7
https://github.com/MaMpf-HD/mampf/blob/b211e4dcd37111a2dacd241668ad764c790bdae6/docker/development/Dockerfile#L1-L7
https://github.com/MaMpf-HD/mampf/blob/b211e4dcd37111a2dacd241668ad764c790bdae6/docker/production/Dockerfile#L1-L7

**Comment by fosterfarrell9** [comment]
I think the current solution with the own dockerfile has worked so far (in devlopment as well as in production), so I do not think that a fundamental change is needed. But if  there are suggestions for other solutions that are robust I am also open.

**Comment by christian-heusel** [comment]
> Adding pdfs together causes an error in the current version (see image). Changing version golang:1.20 back to golang:1.16 seems to fix this issue.

Yes, this is due to an out of date `wasm_exec.js`, I will have a look 👍🏻

File: entrypoint.sh [hunk 1]
```diff
@@ -5,7 +5,7 @@ cd /usr/src/app
 ./initialize.sh &> >(tee -a /usr/src/app/log/initialisation.log)
 
 rm -f tmp/pids/server.pid
-cp /pdfcomprezzor.wasm public/pdfcomprezzor/pdfcomprezzor.wasm
+cp /pdfcomprezzor.wasm /wasm_exec.js public/pdfcomprezzor/
```
Comment (christian-heusel) [inline entrypoint.sh:8]: I really do not know why this is neccessary and would consider it a bug/hack/technical debt or whatever.
It just took me ~30+ min to figure out why my dev container was serving the wrong version of `wasm_exec.js`.

Where should static content like this be saved?

**Comment by christian-heusel** [comment]
> > Adding pdfs together causes an error in the current version (see image). Changing version golang:1.20 back to golang:1.16 seems to fix this issue.
> 
> Yes, this is due to an out of date `wasm_exec.js`, I will have a look 👍🏻

Fixed in https://github.com/MaMpf-HD/mampf/pull/431/commits/8ad0b9375d59a758187998ec8dd7ced9e21de040 and https://github.com/MaMpf-HD/mampf/pull/431/commits/e38d5a5c9aa4abe178d6ff056734030c512f300e ✅

**Pull request merged**